proof/invariant-abstract/SchedContext_AI.thy

```diff
@@ -831,7 +773,7 @@ crunches head_insufficient_loop, handle_overrun_loop
   for valid_replies_pred[wp]: "valid_replies_pred P"
   (wp: crunch_wps ignore: update_sched_context)
 
-lemma refill_budget_check_valid_replies[wp]:
+lemma refill_budget_check_valid_replies_red[wp]:
```

Comment:
Where does the `red` in the name come from?

Reply:
ah, it should be `pred` (ie. what the invariant is about).